pkg/manager/member/pd_failover.go

```diff
@@ -56,16 +56,7 @@ func (f *pdFailover) Failover(tc *v1alpha1.TidbCluster) error {
 		tc.Status.PD.FailureMembers = map[string]v1alpha1.PDFailureMember{}
 	}
 
-	healthCount := 0
-	for podName, pdMember := range tc.Status.PD.Members {
-		if pdMember.Health {
-			healthCount++
-		} else {
-			f.deps.Recorder.Eventf(tc, apiv1.EventTypeWarning, "PDMemberUnhealthy",
-				"%s(%s) is unhealthy", podName, pdMember.ID)
-		}
-	}
-	inQuorum := healthCount > len(tc.Status.PD.Members)/2
+	inQuorum, healthCount := isPDInQuorum(tc, f)
```

Comment:
```suggestion
	inQuorum, healthCount := f.PDInQuorum(tc)
```

Reply:
There is an event recorder needed in the following part.

Reply:
You can call the recorder with `f.deps.`

Reply:
OK, I get it. XD